src/ClientInterface.php

```diff
@@ -12,6 +12,11 @@
  */
 interface ClientInterface
 {
+    /**
+     * The Guzzle major version.
+     */
+    const MAJOR_VERSION = 7;
```

Comment:
That option probably makes the most sense. 👍

Reply:
The only issue is in Guzzle 6 this had type `string`.

Reply:
Ahh, good point. Do you think it'd be better to use a string? I think it's OK to leave as an int.

Reply:
I think that we have to make it a string of we rename it. Otherwise code with strict types enabled will break.

Reply:
Seems like it's not a very useful change at this point, and either the version constant should remain unchanged from guzzle 6, or it should be removed. I still am in favor of a `VERSION` [file in the repo](https://github.com/googleapis/google-cloud-php/blob/master/Storage/VERSION), but that seems like a longshot.

Reply:
IMHO I am in favor of changing it to a string in order to be easier to be used to `version_compare` function of PHP.. but I have mixed feelings if about the naming.. not sure what's the best approach. Changing name will clear indicate that it changed format.. but it will not ease migration...

Reply:
This name is consistent with Symfony's Kernel. 

I dont want to use `VERSION` because people might believe it is the same as in 6.x